features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/components/MarkdownEditor.jsx

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
@hisanhunais It seems like this is a copy of [this component](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/MarkdownEditor.jsx)
Can't we reuse it here ?